src/connectors/nts.ts

```diff
@@ -0,0 +1,36 @@
+export {};
+
+Connector.playerSelector = '.page-live-tracks__body';
+
+Connector.getArtistTrack = () => {
+	let artist = null;
+	let track = null;
+
+	const trackinfo = document.querySelector(
+		'.page-live-tracks__list li:first-child',
+	);
+
+	if (trackinfo) {
+		const artistElement = trackinfo.querySelector(
+			'.page-live-tracks__artist-title',
+		);
+		const trackElement = trackinfo.querySelector(
+			'.page-live-tracks__song-title',
+		);
+		if (artistElement) {
+			artist = artistElement.textContent || '';
+		}
+		if (trackElement) {
+			track = trackElement.textContent || '';
+		}
+	}
+
+	return { artist, track };
+};
```

Comment:
```suggestion
Connector.artistSelector = '.page-live-tracks__list li:first-child .page-live-tracks__artist-title';

Connector.trackSelector = '.page-live-tracks__list li:first-child .page-live-tracks__song-title';
```
We already have internally that should be equivalent to this, I suspect this might work out of the box (though test it first).